system/lib/pthread/emscripten_thread_state.S

```diff
@@ -53,7 +53,8 @@ emscripten_is_main_browser_thread:
   global.get is_main_thread
   end_function
 
-# Semantically the same as testing "!ENVIRONMENT_IS_WEB" in JS
+# Semantically the same as testing "!ENVIRONMENT_IS_WEB" in JS (except for the
```

Comment:
How about just:

```
# Semantically the same as testing "!ENVIRONMENT_IS_WEB && !ENVIRONMENT_IS_AUDIO_WORKLET" since neither the main thread
on the web, nor audio worklets support atomics waits.
```